spatialos/UnrealGDK
# [UNR-1379] Worker entity

**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
See https://github.com/spatialos/UnrealGDK/pull/929

#### Release note
REQUIRED: Add a release note to the `##Unreleased` section of CHANGELOG.md. You can find guidance for writing useful release notes [here](../SpatialGDK/Extras/internal-documentation/how-to-write-good-release-notes.md). Documentation changes are exempt from this requirement.

#### Tests
Tested by running shooter game.

How can this be verified by QA?
Start a game and check the inspector for entities named after worker ID's. There should be one for each server worker and the interest component should contain a query for the GSM entity.

#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?

#### Primary reviewers
@mattyoung-improbable @Vatyx
